elixir/apps/domain/test/domain/policies_test.exs

```diff
@@ -536,24 +527,16 @@ defmodule Domain.PoliciesTest do
 
       attrs = %{actor_group_id: new_actor_group.id}
 
-      assert {:replaced, replaced_policy, replacement_policy} =
-               update_or_replace_policy(policy, attrs, subject)
-
-      assert replaced_policy.resource_id == policy.resource_id
-      assert replaced_policy.actor_group_id == policy.actor_group_id
-      assert replaced_policy.conditions == policy.conditions
-      assert not is_nil(replaced_policy.deleted_at)
-      assert replaced_policy.replaced_by_policy_id == replacement_policy.id
+      assert {:updated, updated_policy} =
+               update_policy(policy, attrs, subject)
 
```

Comment:
```suggestion
      assert updated_policy.id == policy.id
```

Reply:
👍 This suggestion was taken, just moved one line down.